src/Integration.UnitTests/Suppression/IssuesFilterTests.cs

```diff
@@ -41,11 +42,28 @@ public void TestInitialize()
         }
 
         [TestMethod]
-        public void Ctor_NullSonarQubeIssuesProvider_ArgumentNullException()
+        public void Ctor_NullSonarQubeIssuesProvider_ThrowsArgumentNullException()
         {
             Action act = () => new IssuesFilter(null);
 
             act.Should().ThrowExactly<ArgumentNullException>().And.ParamName.Should().Be("sonarQubeIssuesProvider");
         }
+
+        [TestMethod]
+        public void Filter_NullIssues_ThrowsArgumentNullException()
+        {
+
+            Action act = () => testSubject.Filter("c:\\path\\file1.txt", null);
+            act.Should().ThrowExactly<ArgumentNullException>().And.ParamName.Should().Be("issues");
+        }
+
+        [TestMethod]
+        public void Filter_NullFile_DoesNotThrow()
```

Comment:
Future comment: If the file path is irrelevant, then maybe it should be a DataRow input to the test that asserts that sonarQubeIssuesProvider was called with the correct arguments? (will be relevant when there is more implementation).

Reply:
I'll write appropriate tests when there is an implementation. For now I just want enough coverage to pass the QG.